src/components/LoansByCategory/QuickFilters/LocationSelector.vue

```diff
@@ -182,6 +186,7 @@ export default {
 		toggleRegions() {
 			this.showRegions = !this.showRegions;
 			this.selectedRegion = null;
+			this.updateLocation(this.selectedCountries);
```

Comment:
We typically use the pattern of emitting values rather than passing a callback to a component. See the other loan filter components that emit the changed values.

Reply:
yeah we only want to update when the `See {{ totalLoans }} loans` button is clicked which uses this function